docs/getting-started.md

```diff
@@ -0,0 +1,335 @@
+
+
+This tutorial will walk you through the minimum steps needed to run an
+OpenSAFELY-compliant study against "dummy" (random) patient data.  We ask all
+potential collaborators to successfully complete this tutorial, before
+applying to run their project against real data.
+
+This is a first draft of the tutorial, and may have errors or omissions.
+Please don't be afraid to ask questions in our [Q&A
+forum](https://github.com/opensafely/documentation/discussions)!
+
+!!! note "Windows-only!"
+    This version of the tutorial is aimed at Windows users.
+    Mac users should be able to follow along as well, with a few
+    hopefully-obvious alterations; specific instructions are coming soon!
+
+
+## Motivation
+
+OpenSAFELY is designed to allow you to do all of your analytic work on your own
+computer, without ever having to access the real, sensitive, patient-level data.
+
+To use the OpenSAFELY framework effectively, you need to:
+
+*  install a few pieces of software; and
+*  understand a few concepts which might be new to you
+
+Some of the software you'll install is so you can execute code on your computer
+in **exactly the same way** it is run in the secure environment: even a slight
+mismatch in the versions of the software could cause bugs and delays.
+
+OpenSAFELY is also designed to encourage analysts to adopt best-practice
+software development processes, like using the `git` for version control.  If
+you're new to these concepts, there may be quite a lot to learn, and you'll need
+to use further software to work with them. The investment will be worthwhile:
+you'll find your software quality and efficiency will benefit hugely.
+
+This tutorial is, therefore, a very fast tour through all the **essential**
+components required to get up and running.  When you've finished, you'll have
+all the basics in place to continue your learning.
+
+## 1. Install the required software
+
+To develop an OpenSAFELY study on your own computer, you will need to install a
+few things. Most important is the `opensafely` tool; to install this, you must
+(currently) first install the Python programming language.
+
+### Install Python and **opensafely**
+1. [Download and run the Anaconda Python
+   installer](https://docs.anaconda.com/anaconda/install/windows/).
+2. When you've done, to verify your installation, open Anaconda Prompt by
+   clicking Start, search, or selecting Anaconda Prompt from the menu. ![Finding
+   Anaconda Prompt on Windows](./images/win-anaconda-prompt.png)
+3. Install the OpenSAFELY command line tool. Type `pip install opensafely`, and
```

Comment:
Link to how to install `pip` - there are some cases on OS when you need to use `pip3` rather than `pip` which can be a gotcha

Reply:
Oh I assumed Anaconda included `pip`. Is it as simple as `conda install pip`? 

The default Anaconda python is 3.x so doesn't that mean the default `pip` is a python 3 `pip`?

Reply:
nit: I think the wording of this step could be clearer - with my new-user head on, I initially read this as "install the OS command line tool, then type pip etc", but actually you mean "install the OS command tool *by* typing pip etc into the Anaconda prompt"?!

Reply:
From memory, I think you have to install `pip` separately, although this may have changed. I think it's not included because you can install most things via `conda install` instead of `pip install`, although not what we need obviously. 

The `pip3` issue is a separate one for macOS. Because macs always come with Python 2 installed (used in operating system), `pip` defaults to python2. You either have to overwrite this for example via alias or use `pip3`.